server/views/taskList/finalChecksTask.pug

```diff
@@ -7,7 +7,7 @@ div.taskListItem.smallPaddingBottom
                 div.padLeftRight15
                     ul
                         if licenceStatus.decisions.seriousOffence
-                            li.list-bullet.alert#seriousOffenceAnswer The prisoner is under investigation or been charged for a serious offence in custody
+                            li.list-bullet.alert#seriousOffenceAnswer The offender has been referred to the police or other law enforcement agency or to the independent adjudicator (IA) in relation to an alleged offence committed during the current sentence and the matter remains unresolved;
                         else
```

Comment:
Should there be a semicolon at the end?

Reply:
It's what's on the ticket inside the quotes

Reply:
Sorted it out it's not supposed to be there